src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -313,16 +319,16 @@ object ICollection.SyncRoot
 
         #region ICollection<T> members
 
-        public bool Add(T item) => AddIfNotPresent(item);
+        public bool Add(T item) => AddIfNotPresent(item); // Hack so the implementation can be made virtual
 
-        void ICollection<T>.Add(T item) => AddIfNotPresent(item);
+        void ICollection<T>.Add(T item) => Add(item);
```

Comment:
`AddIfNotPresent` is longer than `Add`, plus this method won't have to change if `bool Add` does more than call `AddIfNotPresent` in the future.